provisions/roles/jenkins/master/tasks/ci.yml

```diff
@@ -17,11 +17,17 @@
   when: test
   sudo: yes
 
+- name: wait for 10 seconds
+  pause:
+    seconds: 10
+
```

Comment:
This wait is to let Jenkins create test job or something else? If yes, we are already doing 50 retries with 10 seconds delay. Please add comment with reason we are waiting for 10 seconds.